pkgs/development/python2-modules/gtkme/default.nix

```diff
@@ -0,0 +1,29 @@
+{ pkgs
+, lib
+, pkg-config
+, buildPythonPackage
+, buildPackages
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "gtkme";
+  version = "1.5.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-NIUgnbfcHjbPfsH3CF2Bywo8owrdsi1wqDoMxOa+2U4=";
+  };
+
+  nativeBuildInputs = [ pkg-config pkgs.gobject-introspection ];
+  buildInputs = [ pkgs.python27Packages.pygobject3 pkgs.gtk3 ];
```

Comment:
Actually
```suggestion
  nativeBuildInputs = [ pkg-config gobject-introspection ];
  buildInputs = [ pygobject3 gtk3 ];
```

Reply:
you mean I should add them as input to the derivation.

Reply:
Right. This is the preferred way, as it allows to do individual overrides

Reply:
ok thx for the advice